mvcc/kvstore_test.go

```diff
@@ -522,7 +522,7 @@ func TestHashKVWhenCompacting(t *testing.T) {
 	s := NewStore(b, &lease.FakeLessor{}, nil)
 	defer os.Remove(tmpPath)
 
-	rev := 1000
+	rev := 10000
```

Comment:
it takes an extra second or so with my machine; the txn buffering keeps it cheap